### DISCUSSION TITLE
Chrome Extension

### DISCUSSION THREAD

**Discussion by tedkulp** [shlinkio/discussions#165] [Show and tell]
Last month I made a Chrome extension for saving links to Shlink and ended up publishing it in case it's helpful for anyone else. I know that another extension does exist, but I wanted something that allowed for some options before creating the link.  It can be found [here](https://chrome.google.com/webstore/detail/shlink-it/honjlmannamjohjjmlfkohhcppoelpfc).

Thanks so much for this project! Shlink really is exactly what I was looking for.

**Comment by acelaya** [comment]
Nice!

**Comment by acelaya** [comment]
@tedkulp have you considered opensourcing the extension?

**Reply by tedkulp** [reply]
Sorry, I should've posted the link to the repo.  https://github.com/tedkulp/shlink-it

**Comment by webysther** [comment]
There another one which I use: https://github.com/edward-shen/shlink

https://chromewebstore.google.com/detail/shlink/mgdacpmionfhhogkokjbdeehfnnliajj

Maybe consider add this one in the third-party page.
